source/client.js

```diff
@@ -13,7 +15,11 @@ const pageDetails = parse(window.location.search)
 
 render(
   <div>
```

Comment:
Must be on import then. When I have `if(!document)` or `if(document !== undefined)` before init it fails